packages/core/src/html/includePanelProcessor.ts

```diff
@@ -66,7 +66,7 @@ function _getBoilerplateFilePath(element: MbNode, filePath: string, config: Reco
 function _getSrcFlagsAndFilePaths(element: MbNode, config: Record<string, any>) {
   const isUrl = urlUtil.isUrl(element.attribs.src);
 
-  // We do this even if the src is not a url to get the hash, if any
+  // We do this even if the src is not an url to get the hash, if any
```

Comment:
'a' is actually the correct word as url is pronounced as 'you-ar-el' which leads to the first "sound" letter being 'y'. This article explains in more detail https://owl.purdue.edu/owl/general_writing/grammar/articles_a_versus_an.html

Reply:
Sorry for this. It was highlighted by static analysis and I thought it was a valid typo HAHA